bciers/libs/components/src/form/widgets/CheckboxWidget.tsx

```diff
@@ -24,7 +24,7 @@ const CheckboxWidget: React.FC<WidgetProps> = ({
           }
         />
       </div>
-      <label>{label}</label>
+      <label htmlFor={id}>{label}</label>
```

Comment:
This is a fix to allow clicking on the label to check the box. A nice UX improvement 😎

Reply:
Ooooh I love this! I quickly attempted to do this by adding pointer-event on the whole `div` but that failed and I gave up lol. Thanks for finding the pro solution.